### PR TITLE
Dna 2.0

### PR DESCRIPTION
**Pull request by erulabs**
🌟 Deploy-Node-App 2.0 🌟 

- Complete rewrite
- Powered by Skaffold (writes out a Skaffold.yaml, supports dev, build, deploy etc!)
- Supports other languages! (Python, Ruby for now)
- Meta-modules are now built-in to the repository. Let's keep things simple for now!
- Much more robust handling of file writing and pruning

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,112 +1,81 @@
...
+By default, `deploy-node-app` will write a few files to your directory, and by default files won't be touched if they've been modified.
 
-`deploy-node-app` can deploy to either a Docker or Kubernetes installation locally:
+Simply run `npx deploy-node-app` in your repository. The tool will attempt to prompt you when it needs answers to questions, and do it's best to bootstrap your application. Take a look at [supported languages](https://github.com/kubesail/deploy-node-app/tree/master/src/languages) - we're always looking to add more!
 
-Deploying to local docker-compose:
+# Dependencies
 
-`deploy-node-app local -f compose`
+`deploy-node-app` knows about dependencies! For example, if you install a redis or postgres driver for Node.js, Python, Ruby [and more](https://github.com/kubesail/deploy-node-app/tree/master/src/languages), `deploy-node-appp` will automatically create Redis or Postgres deployments that work with your app!
```
Comment (din0s) [inline README.md]: Little typo that caught my eye!

```suggestion
`deploy-node-app` knows about dependencies! For example, if you install a redis or postgres driver for Node.js, Python, Ruby [and more](https://github.com/kubesail/deploy-node-app/tree/master/src/languages), `deploy-node-app` will automatically create Redis or Postgres deployments that work with your app!
```

File: src/index.js [hunk 2]
```diff
@@ -1,52 +1,74 @@
...
+  require('./languages/php'),
+  require('./languages/python')
+]
+
+let env
+let action
 
 program
-  .arguments('[env]')
+  .name('deploy-to-kube')
```
Comment (din0s) [inline src/index.js]: Isn't this supposed to be `deploy-node-app`?

File: README.md (hunk 1 above)
Comment (erulabs) [reply README.md]: Yes! Fixed, thank you!

File: src/index.js (hunk 2 above)
Comment (erulabs) [reply src/index.js]: Yes it is - We're potentially going to rename this project `deploy-to-kube` due to the fact that this new version should happily support other languages (as long as we can somehow guess their dependencies in a regular way). But we've decided to punt that for now, as Node.js is the only first-class citizen that we've been heavily testing for now. Reverted - thank you for the review!!!

**Review by pastudan — APPROVED** [review]

**Pull request merged**